src/spanningtrees13/kruskal.jl

```diff
@@ -9,11 +9,10 @@ undirected graph `g` with optional distance matrix `distmx` using [Kruskal's alg
 """
 function kruskal_mst end
 # see https://github.com/mauro3/SimpleTraits.jl/issues/47#issuecomment-327880153 for syntax
-@traitfn function kruskal_mst(g::AG::(!IsDirected),
-    distmx::AbstractMatrix{T}=weights(g); minimize=true) where {T <: Real, U, AG <: AbstractGraph{U}}
+@traitfn function kruskal_mst(g::G,
+                              distmx::AbstractMatrix{T}=weights(g); minimize=true) where {T <: Real, G <: AbstractGraph; HasContiguousVertices{G}, !IsDirected{G}}
```

Comment:
I think our convention is to not have spaces between the type annotations inside the `where` clauses. I'd propose a PR but it won't let me edit this file.
```
@traitfn function kruskal_mst(g::G, distmx::AbstractMatrix{T}=weights(g); minimize=true) where {T<:Real, G<:AbstractGraph; HasContiguousVertices{G}, !IsDirected{G}}
```

Reply:
Not sure I see the change you mean here